include/bluetooth/enocean.rst

```diff
@@ -0,0 +1,112 @@
+.. _bt_enocean_readme:
+
+Bluetooth EnOcean
+#################
+
+The Bluetooth EnOcean library allows passive observation of BLE-based wall switches and sensors from `EnOcean`_, such as the Push Button Transmitter Module 215B wall switch module (PTM 215B) and the Easyfit Motion Detector with Illumination Sensor module (EMDCB sensor).
+This means that you can only observe the output of the devices, without sending anything.
+Although the EnOcean devices offer the two-way interaction, this library does not support it.
+
+The library supports different types of commissioning for both wall switches and sensors:
+
+    * radio-based commissioning,
+    * manually inputting data from other commissioning methods, such as NFC or QR codes.
```

Comment:
Is this the preferred punctuation for lists (the comma on the first line, period on the second)?